tests.py

```diff
@@ -1,13 +1,13 @@
-from viscm.gui import *
```

Comment:
Code reformatting should go in a single commit. A commit following that one should include the SHA in the git-blame-ignore (see scipy or skimage).

Reply:
Documented this in `contributing.md`. This look like you expect? 1fec42d , 8aa7bb0

Reply:
Looks good!